dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -2074,6 +2074,21 @@ public void updateTrackedEntityInstances( List<String> teiUids, User user )
             PreparedStatement::execute );
     }
 
+    @Override
+    public void updateEnrollmentsLastUpdatedUserInfo( List<String> enrollmentUids, UserInfoSnapshot userInfoSnapshot )
+    {
+        List<List<String>> uidsPartitions = Lists.partition( Lists.newArrayList( enrollmentUids ), 20000 );
```

Comment:
How did we come up with this batch size? Are we using the same one elsewhere? A constant would help document that 😄

Reply:
@mortenoh